main.js

```diff
@@ -12,9 +12,22 @@ if (isDev) {
 // be closed automatically when the JavaScript object is garbage collected.
 let win
 
-function createWindow () {
+var shouldQuit = app.makeSingleInstance(function(commandLine, workingDirectory) {
```

Comment:
- Add more descriptive comments, to explain this case and the behavior we do.
- Currently, most of the code do not add semi colons by the end of lines